src/templates/operations_problem_templates.jl

```diff
@@ -58,9 +45,30 @@ function template_unit_commitment(; kwargs...)
 
     return template
 end
+"""
+    template_unit_commitment(; kwargs...)
+
+Creates an `OperationsProblemTemplate` with default DeviceModels for a Unit Commitment
+problem.
+
+# Example
+```julia
+template = template_unit_commitment()
+```
+
+# Accepted Key Words
+- `network::Type{<:PM.AbstractPowerModel}` : override default network model settings
+- `devices::Dict{Symbol, DeviceModel}` : override default `DeviceModel` settings
+- `branches::Dict{Symbol, DeviceModel}` : override default `DeviceModel` settings
+- `services::Dict{Symbol, ServiceModel}` : override default `ServiceModel` settings
+"""
+function template_unit_commitment(; kwargs...)
```

Comment:
unit_commitment_template, same for economic dispatch

Reply:
I named these with template first so that they would show up adjacent when looking for the template methods. I like that behavior, but could be convinced otherwise.

Reply:
that's ok with me then. I don't have a strong preference.

Reply:
let's use `make_*` and change the current `make_*` functions to  something else